thinc/tests/strategies.py

```diff
@@ -122,3 +122,22 @@ def arrays_OPFI_BI_lengths(max_B=5, max_P=3, max_F=5, max_I=8):
         )
     )
     return strat
+
+
+__all__ = [
+    "get_ops",
+    "get_model",
+    "get_output",
+    "get_input",
+    "lengths",
+    "shapes",
+    "ndarrays_of_shape",
+    "ndarrays",
+    "arrays_BI",
+    "arrays_BOP",
+    "arrays_BOP_BO",
+    "arrays_BI_BO",
+    "arrays_OI_O_BI",
+    "arrays_OPFI_BI_lengths",
+    "floats",
+]
```

Comment:
I don't think we want to reexport anything in the test suite, especially not function names that kind of shadow core library methods. Is there are a way to avoid this check in the test suite instead? It could be a setting in `setup.cfg` if it needs to be?

Reply:
Strangely, 
```ini
[mypy-thinc.tests.*]
implicit_reexport = False
```
does _not_ work in disabling the implicit-reexport check (it also doesn't work for enabling them). Two alternative workarounds:
- Adding `# type: ignore[attr-defined]` to problematic imports.
- Defining a minimal `__all__` in `thinc.tests.strategies` - the current list could be reduced to `["floats"]` and still pass `mypy thinc --no-implicit-reexport`.

Reply:
You want:

```
[mypy-thinc.tests.*]
implicit_reexport = True
```